lib/src/models/find/data.dart

```diff
@@ -0,0 +1,46 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:path/path.dart';
+
+import '../../utils/utils.dart';
+import '../model.dart';
+
+class FindDataModel extends Model {
+  findStringData(String dataEntry) async {
```

Comment:
I've seen similar code in other models, so should we move this code to the [Model] class?

Reply:
That can be done in a future pr?

Reply:
yeah